packages/icon-button/src/IconButton.test.js

```diff
@@ -1,12 +1,13 @@
 import renderer from "react-test-renderer";
 import React from "react";
 
-import { names } from "../Icons/icons";
+import { names } from "@hig/icons";
 import IconButton from "./IconButton";
+import { default as iconButtonTypes } from "./types";
 
 describe("IconButton", () => {
   it("has type constants", () => {
-    expect(IconButton).toHavePropertyOfConstants("types");
+    expect(iconButtonTypes).toBeDefined();
```

Comment:
This test is no longer about `IconButton`, this should be moved into a new spec for `index.js`. For now, we can simply remove it.